plio/io/io_controlnetwork.py

```diff
@@ -3,8 +3,11 @@
 import pandas as pd
 import numpy as np
 import pvl
+import struct
 
 from plio.io import ControlNetFileV0002_pb2 as cnf
+from plio.io import ControlNetFileHeaderV0005_pb2 as cnh5
+from plio.io import ControlPointFileEntryV0005_pb2 as cnp5
```

Comment:
These are horrible abbreviations, but I couldn't thing of better ones that weren't long.

Reply:
why 'p' and 'h'?

Reply:
point, header